src/components/HomeLayout/Helpline.js

```diff
@@ -1,55 +1,93 @@
 import React, { useState } from "react";
 import { Modal, Row, Col, Button } from "antd";
+import { CarOutlined } from "@ant-design/icons";
+import { travelLinks } from "@utils/constants";
+// import { WomanOutlined } from "@ant-design/icons";
+// import { Spacer } from "@components/Utils";
+// import { helplineLinks } from "@utils/constants";
 
-import { WomanOutlined } from "@ant-design/icons";
-import { Spacer } from "@components/Utils";
-import { helplineLinks } from "@utils/constants";
-
-const HelplineModal = ({ showModal, setShowModal }) => (
+const TravelAssistanceModal = ({ showModal, setShowModal }) => (
   <Modal
-    title={<div>Women&apos;s Helpline</div>}
+    title={<div>Travel Assistance</div>}
     visible={showModal}
     onOk={() => setShowModal(false)}
     onCancel={() => setShowModal(false)}
     footer={null}
   >
     <div style={{ textAlign: "left" }}>
       <div className="modal-txt">
-        <p> Helpline numbers are available in the links below, </p>
-        <span>
-          <a href={helplineLinks.kannada} target="_new">
-            <Button type="primary">ಕನ್ನಡ</Button>
-          </a>
-        </span>
-        <Spacer width={10} />
-        <span>
-          <a href={helplineLinks.english} target="_new">
-            <Button type="primary">English</Button>
-          </a>
-        </span>
+        {travelLinks.map((travel, index) => (
+          <p key={index}><a href={travel.link} target="_new" >
```

Comment:
The right convention is 
<a href="somesite.com" target="_blank" rel="noopener noreferrer">Link</a>
. Please do it the next time you get a change.

Reply:
Sure will do it